libs/contact/src/index.ts

```diff
@@ -1 +1,2 @@
-export {}
+export { DaffContactServiceInterface } from './driver/interfaces/contact-service.interface'
+export { DaffContactUnion } from './models/contact-union';
```

Comment:
You'll need to export the token at some point too.

Reply:
8e2c34ad64c8c3ed5ce65740228937021fab22d0